devito/finite_differences/differentiable.py

```diff
@@ -259,6 +259,12 @@ def __rfloordiv__(self, other):
         from .elementary import floor
         return floor(other / self)
 
+    def safe_inv(self, ref, safe=False):
+        if safe:
```

Comment:
that feels weird. What's the point of a `safe` flag passed to a method called "safe_inv"? Even more if you think it's default to False. Perhaps rename the method as "inv" ?